src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_NUMBER = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[BALL_NUMBER];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery();
+            Ball randomBall = lottery[i].getRandomBall();
+            System.out.println(randomBall);
+        }
```

Comment:
<img width="1010" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/81781479/b2c45c18-06d1-4d1a-9966-906a8d7b9b29">


you dont need to create several instances of Lottery class - it should be just 1, that returns a random ball each time the method `getRandomBall` is called